www/tami/assets/css/style.css

```diff
@@ -131,3 +131,76 @@ body {
       padding-right: 51px;
     }
   }
+
+
+
+  .footer {
+    padding: 49px 16px 48px;
+}
+
+  .footer .menus {
+    display: flex;
+    justify-content: space-between;
+  }
+  .footer .menus .left {
+    display: flex;
+    gap: 40px;
+  }
+  .footer .menus .left .widget {}
+  .footer .menus .left .widget-title {
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 150%;
+    color: #44576C;
+  }
+  .footer .menus .left .widget-content {
+    margin-top: 16px;
+  }
+  .footer .menus .left .widget-content ul {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  }
+  .footer .menus .left .widget-content li {
+    font-size: 14px;
+    line-height: 150%;
+    color: #98A8B9;
+  }
+  .footer .menus .left .widget-content li:not(:first-child) {
+    margin-top: 4px;
+  }
+  .footer .menus .left .widget-content li a {
+    display: flex
+    align-items: center;
+    gap: 2px;
+  }
```

Comment:
<img width="517" alt="image" src="https://user-images.githubusercontent.com/32983425/189894311-074571b1-f862-4bd7-89ed-73b88b1e8a1b.png">
style.css 파일의 174번째줄 부터 스타일이 먹히지 않았다고 알려주는 문구입니다!

확인해보니 `display: flex` 속성을 추가하시고 마지막에 세미콜론(;)을 깜빡하셨네요. 🙂

```suggestion
  .footer .menus .left .widget-content li a {
    display: flex;
    align-items: center;
    gap: 2px;
  }
```

이렇게 수정하면 문제 없이 스타일이 적용될거예요!